### ISSUE TITLE
psqldef syntax error on `REFERENCES table (field)` FK constraint

### ISSUE THREAD

**Issue by AriaFallah** [k0kubun/sqldef#84]
## Problem
I can't define foreign key relationships using REFERENCES on specific fields because I get

```
found syntax error when parsing DDL "CREATE TABLE b (
  b_id INTEGER PRIMARY KEY,
  a_id INTEGER REFERENCES a (a_id),
  a_my_text TEXT NOT NULL REFERENCES a (my_text)
)": syntax error at position 75
```

Running the same code directly in `psql` works though! I'm using the following command to run psqldef

```
PGSSLMODE=disable psqldef -h localhost -U postgres -W verysecure postgres --dry-run < ./db/schema.sql
```

You can find the full grammar here https://www.postgresql.org/docs/9.1/sql-createtable.html if you CTRL-F for "REFERENCES reftable"

### Initial schema
Blank!

### Input DDLs
```sql
CREATE TABLE a (
  a_id INTEGER PRIMARY KEY,
  my_text TEXT UNIQUE NOT NULL
);

CREATE TABLE b (
  b_id INTEGER PRIMARY KEY,
  a_id INTEGER REFERENCES a (a_id),
  a_my_text TEXT NOT NULL REFERENCES a (my_text)
);
```

### Output DDLs
None 😢

**Comment by k0kubun** [comment]
:+1: Fixed the parser and [started releasing v0.8.4](https://github.com/k0kubun/sqldef/actions/runs/405210076).

**Comment by AriaFallah** [comment]
Thank you!
